### PR TITLE
CAN-346, CAN-345, CAN-343, CAN-342, 

### PR DESCRIPTION
**Pull request by ks-rahul**
CAN-344 in progress

### CODE REVIEW

File: src/components/ComponentPages/CreateNewCamp/CampUI.tsx [hunk 1]
```diff
@@ -52,203 +52,201 @@ const CreateNewCampUI = ({
...
-                          placeholder="Parent camp"
-                        ></Select>
-                      </Form.Item>
-                    ) : null}
-                  </Col>
-                </Row>
-                <Row gutter={16}>
-                  <Col xs={24} sm={12}>
+          <Card title={CardTitle} className="can-card-style">
+            <Form
```
Comment (akuks) [inline src/components/ComponentPages/CreateNewCamp/CampUI.tsx]: Can we move this form in a separate component?

File: src/components/ComponentPages/CreateNewTopic/TopicUI.tsx [hunk 2]
```diff
@@ -37,116 +36,111 @@ const CreateNewTopicUI = ({
...
-                          <Option key={nick.id} value={nick.id}>
-                            {nick.nick_name}
-                          </Option>
-                        ))}
-                      </Select>
-                    </Form.Item>
-                  ) : null}
-
+        <Card title={CardTitle} className="can-card-style">
+          <Form
```
Comment (akuks) [inline src/components/ComponentPages/CreateNewTopic/TopicUI.tsx]: Please move form in a separate components

**Review by akuks — CHANGES_REQUESTED** [review]

File: src/components/ComponentPages/CreateNewTopic/TopicUI.tsx (hunk 2 above)
Comment (ks-rahul) [reply src/components/ComponentPages/CreateNewTopic/TopicUI.tsx]: Done.

File: src/components/ComponentPages/CreateNewCamp/CampUI.tsx (hunk 1 above)
Comment (ks-rahul) [reply src/components/ComponentPages/CreateNewCamp/CampUI.tsx]: Done.

**Pull request merged**